packages/component-utilities/src/getColumnExtents.js

```diff
@@ -1,24 +1,39 @@
-import { tidy, summarize, min, max, median } from '@tidyjs/tidy';
+import { tidy, summarize, min, max, median, mean, n, nDistinct, sum } from '@tidyjs/tidy';
 
 /**
  *
  * @param {Record<string, unknown>[]} data
  * @param {string} columnName
- * @returns {{ min?: number, max?: number, median?: number, maxDecimals: number, unitType: string }}
+ * @param {boolean} [isNumeric]
+ * @returns {{ min?: number, max?: number, median?: number, mean?: number, count?: number, countDistinct?: number, sum?: number, maxDecimals: number, unitType: string }}
  */
-export function getColumnUnitSummary(data, columnName) {
+export function getColumnUnitSummary(data, columnName, isNumeric = true) {
 	const seriesExtents = tidy(
 		data,
-		summarize({ min: min(columnName), max: max(columnName), median: median(columnName) })
+		isNumeric
+			? summarize({
+					count: n(columnName),
+					countDistinct: nDistinct(columnName),
+					min: min(columnName),
+					max: max(columnName),
+					median: median(columnName),
+					mean: mean(columnName),
+					sum: sum(columnName)
+			  })
+			: summarize({ count: n(columnName), countDistinct: nDistinct(columnName) })
```

Comment:
Is this going to run all aggs for all numeric columns, regardless if they are actually displayed?

Reply:
Yes - it was already calculating the median, min & max for all columns as we use that information for automatically formatting column data.
I've added `count`, `countDistinct`, `mean` & `sum` to the same call.

I'm unclear on performance implications but I assume they will not be significantly more expensive that the existing functions